src/plugins/home/server/tutorials/instructions/onprem_cloud_instructions.ts

```diff
@@ -6,9 +6,10 @@
  * Side Public License, v 1.
  */
 
+import { memoize } from 'lodash';
 import { i18n } from '@kbn/i18n';
 
-export const createTrycloudOption1 = () => ({
+export const createTrycloudOption1 = memoize(() => ({
```

Comment:
Same, called for each tutorial, even though we don't mutate them in any way. memoize ease the CPU and GC utilization.